content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -610,6 +611,7 @@ parameters:
   unmanaged disk in the same resource group as the cluster. When `kind` is 
   `managed`, all managed disks are created in the same resource group as 
   the cluster.
+* `resourceGroup`: Specify the resource group in which the Azure disk will be created. It must be an existing resource group name. If it is unspecified, the disk will be placed in the the same resource group as the current Kubernetes cluster.
```

Comment:
This line is wrapped differently from the ones before it. I recommend following the wrapping that's already set.

Reply:
Agreed with @sftim, would be best to follow the line-wrapping that is in place here.

Reply:
For sure, thank you!

Reply:
@mcurtiscmu I wonder if you've got time to tweak this pull request?